include/iarc7_fc_comms/CommonFcComms.hpp

```diff
@@ -478,13 +518,39 @@ void CommonFcComms<T>::reconnect() {
     if(status == FcCommsReturns::kReturnOk)
     {
         ROS_INFO("iarc7_fc_comms: Succesful reconnection to flight controller");
+        calibrateAccelerometer();
```

Comment:
This function handles its failures. Since we don't use exceptions there isn't anything to check